Code/Editor/Dialogs/PythonScriptsDialog.cpp

```diff
@@ -79,6 +79,8 @@ CPythonScriptsDialog::CPythonScriptsDialog(QWidget* parent)
         GetGemSourcePathsVisitor(AZ::SettingsRegistryInterface& settingsRegistry)
             : m_settingsRegistry(settingsRegistry)
         {}
+
+        using AZ::SettingsRegistryInterface::Visitor::Visit;
```

Comment:
The using to bring the base class functions of AZ::SettingsRegistryInterface::Visitor into scope isn't needed because `AZ::SettingsRegistryInterface::Visitor` isn't a templated class

Reply:
there is a warning that is issued, if you override one member, you end up hiding the others from the base class. This using prevents the hiding of those base class members.
In the case of `Visit` here, I was wondering if we want to hide them, but hiding them wont give you much because you can still call those methods through the base class (which is how this is being used).